zebra/interface.c

```diff
@@ -216,6 +217,9 @@ static int if_zebra_delete_hook(struct interface *ifp)
 
 		rtadv_if_fini(zebra_if);
 
+		bond = &zebra_if->bond_info;
+		list_delete(&bond->mbr_zifs);
```

Comment:
I think we should call `zebra_l2if_update_bond(ifp, false);` instead. It does NULL dereference check, etc.